src/Compilers/Extension/CompilerExtension.csproj

```diff
@@ -98,6 +99,10 @@
     <Reference Include="System.Drawing" />
     <Reference Include="System.Windows.Forms" />
     <Reference Include="System.Xml" />
+    <ProjectReference Include="..\..\Scripting\CSharp\CSharpScripting.csproj">
+      <Project>{066f0dbd-c46c-4c20-afec-99829a172625}</Project>
+      <Name>CSharpScripting</Name>
+    </ProjectReference>
```

Comment:
Is that sufficient to get it included in the VSIX?  Some of the references below have additional items related to VSIX inclusion.  It's possible it's implicitly included though.  Did you check the built VSIX? 

@jasonmalinowski do we need to do anything else here?